### ISSUE TITLE
Bun doesn't seem to work on my computer.

### ISSUE THREAD

**Issue by ulisesvina** [oven-sh/bun#197]
Hello, I've been trying to using Bun on my computer for a month now with no luck in neither macOS (Hackintosh) or Linux. This seems to be an issue ONLY with the `bun create` command, as `bun ./file` does work on both OS'es. Apparently, it's a CPU instruction error, it goes as follows:
```
Fetching package.json...
InstructionError at 41572373


–––– bun meta ––––
Bun v0.0.83 Linux x64 #1 ZEN SMP PREEMPT_DYNAMIC Thu, 16 Jun 2022 20:40:47 +0000
CreateCommand:
HTTP: 1
Elapsed: 208ms | User: 6ms | Sys: 26ms
RSS: 0.27GB | Peak: 23.00MB | Commit: 0.27GB | Faults: 20
–––– bun meta ––––

Ask for #help in https://bun.sh/discord or go to https://bun.sh/issues
```
Right now I don't think I'll be able to test again on macOS, but as this is occurring on Linux too we can discard it's some weird Hackintosh drivers issue. I'm using Arch Linux with kernel 5.18.5-zen-1.

**Comment by ulisesvina** [comment]
Recently, I have discovered that this problem occurs across all the commands that involve package managing. So `bun install`, `bun add` and `bun remove` are afected too.

**Comment by ulisesvina** [comment]
This problem seems to be exclusive to my Thinkpad. I tested on another computer (same arch.x86_64 and gen. Haswell) and it seems to be working fine

![image](https://user-images.githubusercontent.com/20508563/181296900-0ca97819-a6aa-43dc-b365-7914e6e76a18.jpeg)
![image](https://user-images.githubusercontent.com/20508563/181296918-6227d5b8-400d-4229-aa95-6c3b456c8061.jpeg)

**Comment by ulisesvina** [comment]
Fixed in v0.1.6.
